files/standalone.xml

```diff
@@ -0,0 +1,349 @@
+<?xml version='1.0' encoding='UTF-8'?> 
```

Comment:
We will have the same amount of maintenance as before. This file replaces https://github.com/puppetlabs/razor-torquebox/blob/master/standalone.xml. So rather than maintaining that, we will be maintaining this. That is, unless we release any updates to razor with the old path format that require a new torquebox package, which I sincerely hope we do not